Razor Class Library template supports RP/MVC or components (#16684)
aspnet/AspNetCore.Docs: aspnetcore/blazor/class-libraries.md

   1. Right-click the app project. Select **Add** > **Reference**.
   1. Select the RCL project. Select **OK**.

> [!NOTE]

> > If the **Support pages and views** check box is selected when generating the RCL from the template, then also add a *_Imports.razor* file to root of the generated project with the following contents to enable Razor component authoring:

Author: ```suggestion
> If the **Support pages and views** check box is selected when generating the RCL from the template, then also add an *_Imports.razor* file to root of the generated project with the following contents to enable Razor component authoring:
```

   1. Right-click the app project. Select **Add** > **Reference**.
   1. Select the RCL project. Select **OK**.

> [!NOTE]

> > If the **Support pages and views** check box is selected when generating the RCL from the template, the generated project doesn't include the *_Imports.razor* file with the following contents:

Reviewer: ```suggestion
> If the **Support pages and views** check box is selected when generating the RCL from the template, then also add a *_Imports.razor* file to root of the generated project with the following contents to enable Razor component authoring:
```

>
> ```razor
> @using Microsoft.AspNetCore.Components.Web
> ```
>
> Manually add the file the root of the generated project.

> >

Author: ```suggestion
```

> > This issue will be addressed in a future release of ASP.NET Core. For more information, see [RCL template either supports MVC OR supports Components (dotnet/aspnetcore #17870)](https://github.com/dotnet/aspnetcore/issues/17870).

Reviewer: I don't think we need to link to the issue here in the docs. This isn't a blocking bug. Let's remove this line.

> > This issue will be addressed in a future release of ASP.NET Core. For more information, see [RCL template either supports MVC OR supports Components (dotnet/aspnetcore #17870)](https://github.com/dotnet/aspnetcore/issues/17870).

Author: ```suggestion
```

   dotnet new razorclasslib -o MyComponentLib1
   ```

   > [!NOTE]

>    > If the `-s|--support-pages-and-views` switch is used when generating the RCL from the template, then also add a *_Imports.razor* file to root of the generated project with the following contents to enable Razor component authoring:

Author: ```suggestion
> If the `-s|--support-pages-and-views` switch is used when generating the RCL from the template, then also add an *_Imports.razor* file to root of the generated project with the following contents to enable Razor component authoring:
```

   dotnet new razorclasslib -o MyComponentLib1
   ```

   > [!NOTE]
   > If the `-s|--support-pages-and-views` switch is used when generating the RCL from the template, then also add a *_Imports.razor* file to root of the generated project with the following contents to enable Razor component authoring:
   >
   > ```razor
   > @using Microsoft.AspNetCore.Components.Web
   > ```
   >
   > Manually add the file the root of the generated project.

>    >

Author: ```suggestion
```

>    > This issue will be addressed in a future release of ASP.NET Core. For more information, see [RCL template either supports MVC OR supports Components (dotnet/aspnetcore #17870)](https://github.com/dotnet/aspnetcore/issues/17870).

Author: ```suggestion
```

>    

Author: ```suggestion
```

   dotnet new razorclasslib -o MyComponentLib1
   ```

   > [!NOTE]

>    > If the `-s|--support-pages-and-views` switch is used when generating the RCL from the template, the generated project doesn't include the *_Imports.razor* file with the following contents:

Reviewer: Same suggested updates as above

Author: ```suggestion
> If the `-s|--support-pages-and-views` switch is used when generating the RCL from the template, then also add a *_Imports.razor* file to root of the generated project with the following contents to enable Razor component authoring:
```
